Description: Enable identity_users_panel
 Otherwise, there's too many errors.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-02-18

--- horizon-9.0.0~b2+2016.02.18.git.cda9604792.orig/openstack_dashboard/enabled/_3031_identity_users_panel.py
+++ horizon-9.0.0~b2+2016.02.18.git.cda9604792/openstack_dashboard/enabled/_3031_identity_users_panel.py
@@ -24,7 +24,7 @@ PANEL_GROUP = 'default'
 PANEL = 'ngusers'
 
 # If set to True, this settings file will not be added to the settings.
-DISABLED = True
+DISABLED = False
 
 # Python panel class of the PANEL to be added.
 ADD_PANEL = 'openstack_dashboard.dashboards.identity.ngusers.panel.NGUsers'
